praw/models/reddit/subreddit.py

```diff
@@ -2933,7 +2933,7 @@ def create(self, name, content, reason=None, **other_settings):
         :param reason: (Optional) The reason for the creation.
         :param other_settings: Additional keyword arguments to pass.
 
-        To create the wiki page ``'praw_test'`` in ``'/r/test'`` try:
+        To create the wiki page ``'praw_test'`` in ``'r/test'`` try:
```

Comment:
The use of single quotes around these values seems a little inconsistent. While I may have added them (don't feel like verifying) they certainly seem a little unnecessary within the inline code blocks.

Reply:
I agree. I'll fix them!